src/main/ui/src/components/SearchByName.jsx

```diff
@@ -48,14 +48,12 @@ return (
       })
       .map((park, key) => {
         return (
-            <div>
-          <p key={key}>
+            <div key={key}>
             <h2>{park.fullName}</h2>
             <br />
             {park.states}
             <br />
             {park.description}
-          </p>
           </div>
         )
         })
```

Comment:
SearchName is in the SearchByName folder. You should delete SearchName and move SearchByName into the SearchByName folder